SparkPost/2web2ui
# AC-1864 - snippets edit form, copy field id/for

### What Changed

- Code Example copy field in the snippets edit form needed an id

### How To Test

- http://localhost:3100/snippets/edit/snippet-create - edit a snippet
Make sure the Code Example field has an id/for combo and the label click focuses the input.
![image](https://user-images.githubusercontent.com/4204806/111693828-21e50700-87ff-11eb-82ad-9bad54ef8341.png)


### To Do
